react-frontend/src/components/Items/CreateItem.js

```diff
@@ -55,6 +60,9 @@ class CreateItem extends Component {
             image,
```

Comment:
this can be removed if we are changing the model to use img

Reply:
Cleaned this up.